libmariadb/mariadb_lib.c

```diff
@@ -1104,7 +1105,8 @@ ma_set_connect_attrs(MYSQL *mysql)
 
   rc+= mysql_optionsv(mysql, MYSQL_OPT_CONNECT_ATTR_ADD, "_client_name", "libmariadb")
        + mysql_optionsv(mysql, MYSQL_OPT_CONNECT_ATTR_ADD, "_client_version", MARIADB_PACKAGE_VERSION)
-       + mysql_optionsv(mysql, MYSQL_OPT_CONNECT_ATTR_ADD, "_os", MARIADB_SYSTEM_TYPE);
+       + mysql_optionsv(mysql, MYSQL_OPT_CONNECT_ATTR_ADD, "_os", MARIADB_SYSTEM_TYPE)
+       + mysql_optionsv(mysql, MYSQL_OPT_CONNECT_ATTR_ADD, "_host_name", mysql->host);
```

Comment:
Connection attributes are client attributes. Why do you need to send the server name to the server (mysql->host) ?